recipes/lixtools/meta.yaml

```
@@ -33,9 +33,9 @@ requirements:
 test:
   imports:
     - lixtools
-  commands:  # [not win]
-    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "plate_label_template.html"))'  # [not win]
-    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "template_report.ipynb"))'  # [not win]
+  commands:
+    - python -c "import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "plate_label_template.html"))"
+    - python -c "import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "template_report.ipynb"))"
```

The double quotes inside should be replaced with single quotes or escaped to properly format the command.

```suggestion
    - python -c "import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename('lixtools', 'plate_label_template.html'))"
    - python -c "import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename('lixtools', 'template_report.ipynb'))"
```